### PR TITLE
Added a generalised kNN(works for any number of features)

### PR DESCRIPTION
**Pull request by euler16**
kNN classifier modeled on the interface shown in the video. Works for any number of features, uses numpy

### CODE REVIEW

File: src/part_5/kNNClassifier.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+			# for testPoint in testFeatures:
+			# 	prediction.append( closest( testPoint ) )
+			# return numpy.array(prediction)			
+			prediction = numpy.array( [closest(point) for point in testFeatures] )
+			return prediction 
+
+
+if __name__ == '__main__':
+
+	from check import CheckClassifier
```
Comment (SZanlongo) [inline src/part_5/kNNClassifier.py:45]: No 'check' module

File: src/part_5/kNNClassifier.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	#IRISDATASET
+
+
+	#IRIS DATASET
+	feature_set = numpy.loadtxt('Fisher.csv',delimiter = ',',skiprows = 1,usecols = (1,2,3,4))
```
Comment (SZanlongo) [inline src/part_5/kNNClassifier.py:50]: No accompanying csv file
May want to replace with appropriate

>  iris = load_iris()

imports.

**Review by SZanlongo — COMMENTED** [review]
Missing accompanying files 'check' and csv dataset (or corresponding scikit-provided datasets)

**Comment by euler16** [comment]
Added the missing files (iris dataset, titanic dataset and check.py)

**Pull request merged**